Changelogs for Digital Twin Manager Plugins 1.4.0 (#2753)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020-2-0-display-device-related-custom-properties-in-asset-properties-widget.md

---
date: ""

> title: "Display device-related custom properties in Asset Properties  widget"

Reviewer: ```suggestion
title: "Device-related custom properties displayed in Asset properties widget"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-QHu1GdukP"
      label: "Feature"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1014"
version: "1020.2.0"
---

> Previously, the select property dialog in the asset property widget, displayed only the default and computed properties of a device. With this change, any custom fragments added to the device will also be displayed.

Reviewer: ```suggestion
Previously, the select property dialog in the "Asset properties" widget only displayed the default and computed properties of a device. With this change, all custom fragments added to the device are also displayed.
```
